database/db_build.js

```diff
@@ -1,23 +1,38 @@
-const career = require('./career_model.js');
-const db = require('./db_connection.js');
 const careers = require('./seed_data.js');
-// const mongoose = require('mongoose');
+const mongoose = require('mongoose');
 
-db.once('open', () => {
-  console.log('we are connected to DB');
-  career.collection.drop(); // once app is open, drop schema career
-
-  careers.forEach((career) => {
-    career.save((error, result) => {
-      if (error) {
-        // render error hbs
-        return console.log(error);
-      }
-    });
+var dataLength = 0;
+for (var i = 0; i < careers.length; i++) {
+  careers[i].save((error, result) => {
```

Comment:
I did some research and found out that save and insert are essentially very similar, see [this](https://alvinalexander.com/source-code/scala/mongodb-whats-difference-between-save-and-insert-mongodb). If this is true then we could get rid of the for loop and use insertMany() instead.